src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/ProjectSystem/VS/Xproj/MigrationReport.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Xproj
+{
+    /// <summary>
+    /// Migration report holder. Copied from
+    /// https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.ProjectJsonMigration/Report/MigrationReport.cs
+    /// </summary>
+    internal class MigrationReport
+    {
+        [JsonProperty]
+        public List<ProjectMigrationReport> ProjectMigrationReports { get; set; }
```

Comment:
Missing private setter here. 

(Also need to change property type too as noted elsewhere.)